packages/govuk-frontend/src/govuk/components/header/_index.scss

```diff
@@ -98,7 +113,7 @@
     font-size: 30px; // We don't have a mixin that produces 30px font size
     line-height: 1;
 
-    @include govuk-media-query($from: tablet) {
+    @include govuk-media-query($from: desktop) {
```

Comment:
@owenatgov Having an parent link with `display: inline` didn't help child alignment, but think it's a bug from https://github.com/alphagov/govuk-frontend/pull/2549 where `display: inline` was added at tablet size but the product name didn't wrap until desktop